app/src/main/res/values/strings.xml

```diff
@@ -457,6 +457,7 @@ Upload your first media by touching the camera or gallery icon above.</string>
     <item quantity="other">Receiving shared content. Processing the images might take some time depending on the size of the images and your device</item>
   </plurals>
 
+  <string name="recieving_content">Receiving shared content. Processing the image might take some time depending on the size of the image and your device</string>
```

Comment:
This string should already exists. `receiving_shared_content `

Reply:
Yes I'll remove that